File: JavaCompiler/src/app/core/models/project/file.ts
export class File {

    public name:string;
    public package:string;
    public code:string;
    public id:string;

    constructor(name:string,id:string,packet:string,code:string){
        this.name = name;
        this.id = id;
        this.package = packet;
        this.code = code;
    }

}

File: JavaCompilerServer/src/main/java/com/hluther/javacompiler/modelos/proyecto/File.java
package com.hluther.javacompiler.modelos.proyecto;

import com.fasterxml.jackson.annotation.JsonTypeName;
import com.fasterxml.jackson.databind.annotation.JsonNaming;

public class File {

    private String name;
    private String id;

    private String package_;
    private String code;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPackage_() {
        return package_;
    }

    public void setPackage_(String package_) {
        this.package_ = package_;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }
}
